### PR TITLE
QPPSF-4832 Correcting 2019 WI Benchmarks

### PR DESCRIPTION
**Pull request by Jpec07**
#### Motivation for change

The WI benchmarks included in the previous pull request for this ticket were incorrect. After collaborating with BAs, we were able to ensure that the right data was included.

#### What is being changed

Two WI measure benchmarks have been removed as they are not included in 2019 data. Two WI measure benchmarks have been added as they are included in 2019, but were not in the previous year.

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [x] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [x] Documentation updated
* [x] Unit tests added/passing
* [x] Verified working locally

##### Associated JIRA tickets:
* https://jira.cms.gov/browse/QPPSF-4832

### CODE REVIEW

**Comment by NateTheGreatt** [comment]
thanks for the fix sir 👍

**Comment by kyeah** [comment]
Thanks @Jpec07! Since we generate the `benchmarks.json` from the `staging/benchmarks/benchmarks.csv` file, could you make the appropriate changes from this and #271 in the csv? That way if we ever need to regenerate the json entirely, we won't lose these benchmarks.

See the documentation [here](https://github.com/CMSgov/qpp-measures-data/blob/e9d848990e1ea3090cfc5459121cf407a6e23936/CONTRIBUTING.md#additional-benchmarks) under Additional Benchmarks on how to regenerate the json file. This allows us to check some validations (e.g. no dupes). Otherwise, the json should look the same as what you have now.

**Comment by Jeffreyher** [comment]
@Jpec07 If it helps, there should be an existing CSV file I think, just add the new benchmarks to that CSV and run the script!

**Comment by Jpec07** [comment]
All set. Windows' implementation of bash gave me some issues with the script, but I managed to get it working.

File: benchmarks/2019.json [hunk 1]
```diff
@@ -2465,6 +2447,24 @@
       100
     ]
   },
+  {
+    "measureId": "226",
```
Comment (Jeffreyher) [inline benchmarks/2019.json]: Is this suppose to be '336'?

File: benchmarks/2019.json [hunk 2]
```diff
@@ -1133,24 +1151,6 @@
       100
     ]
   },
-  {
-    "measureId": "111",
```
Comment (Jeffreyher) [inline benchmarks/2019.json]: Are we ok w/o having '111' w/ submissionMethod: cmsWebInterface?

File: benchmarks/2019.json [hunk 3]
```diff
@@ -1511,24 +1511,6 @@
       100
     ]
   },
-  {
-    "measureId": "128",
```
Comment (Jeffreyher) [inline benchmarks/2019.json]: Are we ok w/o having '128' w/ submissionMethod: cmsWebInterface?

Comment (Jpec07) [reply benchmarks/2019.json]: Yes. Natalia did a very comprehensive analysis of the cmsWebInterface changes for Y3. See here for more details: https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPPRACTICE&title=Year+3+Changes+in+Web+Interface

File: benchmarks/2019.json (hunk 2 above)
Comment (Jpec07) [reply benchmarks/2019.json]: Yes. Natalia did a very comprehensive analysis of the cmsWebInterface changes for Y3. See here for more details: https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPPRACTICE&title=Year+3+Changes+in+Web+Interface

**Comment by Jeffreyher** [comment]
Hey @Jpec07 looks good! I added a couple comments on whether a couple of benchmarks that you are expecting to be added aren't actually being added. Also If we could get a confirmation that the 8 WI measures are indeed the entire set that needs to be added. Thanks!

File: benchmarks/2019.json (hunk 1 above)
Comment (Jpec07) [reply benchmarks/2019.json]: Nope, 226. Natalia did a very comprehensive analysis of the cmsWebInterface changes for Y3. See here for more details: https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPPRACTICE&title=Year+3+Changes+in+Web+Interface

**Comment by Jpec07** [comment]
Hi Jeff,

Thanks for the quick review! I worked closely with one of our BAs, Natalia, to make sure that everything was aligned. She documented everything on Confluence, which I used as my source of truth when making these changes: https://confluence.cms.gov/pages/viewpage.action?spaceKey=QPPPRACTICE&title=Year+3+Changes+in+Web+Interface

**Pull request merged**